packages/core/src/services/documentLogs/computeDocumentLogsWithMetadata.ts

```diff
@@ -70,53 +73,55 @@ export function computeDocumentLogsWithMetadataQuery(
 ) {
   const { scope } = getRepositoryScopes(workspaceId, db)
   const offset = calculateOffset(page, pageSize)
-  const filteredSubQuery = db
-    .select({
-      id: scope.id,
-      error: scope.error,
-    })
-    .from(scope)
-    .innerJoin(commits, eq(commits.id, scope.commitId))
+  const filteredSubQuery = scope
     .where(
-      getCommonQueryConditions({ scope, documentUuid, draft, allowAnyDraft }),
+      getCommonQueryConditions({
+        scope: documentLogs,
+        documentUuid,
+        draft,
+        allowAnyDraft,
+      }),
     )
-    .orderBy(desc(scope.createdAt))
+    .orderBy(desc(documentLogs.createdAt))
     .limit(parseInt(pageSize))
     .offset(offset)
     .as('filteredDocumentLogsSubQuery')
 
   const aggregatedFieldsSubQuery = db
     .select({
-      id: scope.id,
+      id: filteredSubQuery.id,
       tokens: sum(providerLogs.tokens).mapWith(Number).as('tokens'),
       duration: sum(providerLogs.duration).mapWith(Number).as('duration_in_ms'),
       costInMillicents: sum(providerLogs.costInMillicents)
         .mapWith(Number)
         .as('cost_in_millicents'),
     })
-    .from(scope)
-    .innerJoin(filteredSubQuery, eq(filteredSubQuery.id, scope.id))
-    .leftJoin(providerLogs, eq(providerLogs.documentLogUuid, scope.uuid))
-    .groupBy(scope.id)
+    .from(filteredSubQuery)
+    .leftJoin(
+      providerLogs,
+      eq(providerLogs.documentLogUuid, filteredSubQuery.uuid),
+    )
+    .groupBy(filteredSubQuery.id)
     .as('aggregatedFieldsSubQuery')
 
   return {
     scope,
     baseQuery: db
       .select({
-        ...scope._.selectedFields,
+        ...filteredSubQuery._.selectedFields,
         commit: getTableColumns(commits),
         tokens: aggregatedFieldsSubQuery.tokens,
         duration: aggregatedFieldsSubQuery.duration,
         costInMillicents: aggregatedFieldsSubQuery.costInMillicents,
       })
-      .from(scope)
-      .innerJoin(commits, eq(commits.id, scope.commitId))
+      .from(documentLogs)
+      .innerJoin(filteredSubQuery, eq(filteredSubQuery.id, documentLogs.id))
+      .innerJoin(commits, eq(commits.id, filteredSubQuery.commitId))
       .innerJoin(
         aggregatedFieldsSubQuery,
-        eq(aggregatedFieldsSubQuery.id, scope.id),
+        eq(aggregatedFieldsSubQuery.id, filteredSubQuery.id),
       )
-      .orderBy(desc(scope.createdAt)),
+      .orderBy(desc(documentLogs.createdAt)),
```

Comment:
So `scope` is not used now here? only is exported. Maybe is not used outside and we can remove it